src/Symfony/Component/Routing/RequestContext.php

```diff
@@ -48,10 +49,12 @@ class RequestContext
      * @param int    $httpsPort   The HTTPS port
      * @param string $path        The path
      * @param string $queryString The query string
+     * @param string $basePath    The base path
```

Comment:
True, however for being a _request_ context it seemed a reasonable feature to me. And more important, allows this fix :))